src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,40 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return await _userService.CheckCanManageAllTaxonomy()
+                .OnSuccess(_ => _context.Themes
+                    .Where(theme => theme.Title.StartsWith("UI test theme") || theme.Title.StartsWith("Seed theme"))
```

Comment:
No action to be taken on this - but just noting that the performance tests also create a theme - but it is named "UI test theme - Performance tests" (found in `tests/performance-tests/src/tests/testData.ts`) and so would be cleaned up by this.